### ISSUE TITLE
Beanstalk driver not compatible with current version of Pheanstalk

### ISSUE THREAD

**Issue by rpacker** [Codeception/module-queue#1]
#### What are you trying to achieve?
I try to connect to beanstalk with the new syntax: 
$this->queue = Pheanstalk::create($config['host'], $config['port'], $config['timeout']);

#### What do you get instead?
It wants you to use the old syntax:
$this->queue = new Pheanstalk($config['host'], $config['port'], $config['timeout']);

```bash
  [TypeError] Argument 1 passed to Pheanstalk\Pheanstalk::__construct() must be an instance of Pheanstalk\Connection, string given, called in /home/robert/code/hermes/vendor/codeception/codeception/src/Codeception/Lib/Driver/Beanstalk.php on line 18  
```

### Details

* Codeception version:
Codeception 3.1.2
* PHP Version:
PHP 7.2.24-
* Operating System:
Linux Mint

**Comment by SamMousa** [comment]
The interface that needs to be implemented seems simple enough. I'll implement a new class and have the queue module decide on which to use based on the version of Pheanstalk that's available.

**Comment by Naktibalda** [comment]
Fixed by #2 
Released as 1.1.0
